recipes/meshplot/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "meshplot" %}
+{% set version = "v0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/skoch9/meshplot/archive/{{ version }}.tar.gz
+  sha256: 4637d49fecfd5a61bfdbfc22c0b23f7a698ae8336e02296b1f442232559c0566
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+    - pythreejs
+    - numpy
+    - matplotlib
+    - ipywidgets
+  run:
+    - python
+    - pythreejs
+    - {{ pin_compatible('numpy') }}
```

Comment:
This can probably just be `numpy` since you are not compiling against numpy's C-API.